libs/api/domains/national-registry/src/lib/dto/FamilyCorrectionInput.input.ts

```diff
@@ -3,10 +3,6 @@ import { IsString } from 'class-validator'
 
 @InputType()
 export class FamilyCorrectionInput {
-  @Field()
-  @IsString()
-  ssn!: string
-
   @Field()
   @IsString()
   ssnChild!: string
```

Comment:
ssn should be nationalID (the common name for kennitala in island.is)

Reply:
Additionally, what is the scope of the project here?

1. Can a person file a correction for any child?
2. Can the person only file a correction for a child that is signed as his/her family member?

Reply:
* Yes good point, I will change it to nationalId.

The scope should be that only a parent/legal guardian of a child can request a correction for their child's info.